### ISSUE TITLE
RFE: implement version command or argument

### ISSUE THREAD

**Issue by pvalena** [psss/tmt#445]
```
$ tmt version
Usage: tmt [OPTIONS] COMMAND [ARGS]...
Try 'tmt --help' for help.

Error: No such command 'version'.

$ tmt --version
Usage: tmt [OPTIONS] COMMAND [ARGS]...
Try 'tmt --help' for help.

Error: no such option: --version
```

**Comment by martinky82** [comment]
Version is currently stored only in tmt.spec. I wonder, would implementing this feature require storing the version somewhere in the code itself?

Brainstorming: in the code ifself (base.py), there would an artificial value that would translate into 'git snapshot', 'not a released version' or whatever in the message printed by --version. During build, this artificial value would be overwritten by %version (simple sed -i).

Thoughts?

**Comment by psss** [comment]
The new `--version` option implemented in #561.

**Comment by pvalena** [comment]
Thanks!
